src/utils.ts

```diff
@@ -1840,27 +1870,52 @@ const MARKERS = ['+', 'crux', 'mark'];
 const CROSSES = ['x', 'cross', 'oblique'];
 const SQUARES = ['square'];
 const TRIANGLES = ['^', 'triangle', 'tri'];
+const STROKE_STYLE_OFFSET = 15;
 export class newPoint2D extends newShape {
   public path: Path2D;
   public center: Vertex;
 
   constructor(
     x: number = 0,
     y: number = 0,
-    private _size: number = 2,
-    private _marker: string = 'circle',
-    private _markerOrientation: string = 'up',
-    color?: string
+    protected _size: number = 2,
+    protected _marker: string = 'circle',
+    protected _markerOrientation: string = 'up',
+    fillStyle?: string,
```

Comment:
Prefer using null as default instead of coercition operator

Reply:
Done where i found it